src/TimeoutMigrationTool/MigrationRunner.cs

```diff
@@ -61,7 +66,7 @@ public async Task Run(DateTime cutOffTime, EndpointFilter endpointFilter, IDicti
                     continue;
                 }
 
-                logger.LogInformation($"Starting migration for {endpointToMigrate.EndpointName}, {endpointToMigrate.NrOfTimeouts}");
+                logger.LogInformation($"Starting migration for {endpointToMigrate.EndpointName}, {endpointToMigrate.NrOfTimeouts} -- {DateTime.Now}");
```

Comment:
How about we make the `--verbose` option include the timings for all logstatements to avoid embedding the time into all strings? 

Ie to add an if here https://github.com/Particular/TimeoutMigrationTool/blob/master/src/TimeoutMigrationTool/ConsoleLogger.cs#L30